### ISSUE TITLE
[Bug] Following build instructions for static library leads to error

### ISSUE THREAD

**Issue by dnns92** [openvinotoolkit/openvino#9606]
##### System information (version)

- OpenVINO=> 2020.4
- Operating System / Platform => Windows 64 Bit
- Compiler => VS2019
- Problem classification => Installation

##### Detailed description
I was trying to build a static library of openvino, following this tutorial: https://github.com/openvinotoolkit/openvino/wiki/StaticLibraries/
Following it leads to the error message: 
~~~bash
CMake Error at cmake/developer_package/vs_version/vs_version.cmake:41 (message):
  ie_add_vs_version_file can work only with dynamic libraries
Call Stack (most recent call first):
  ngraph/core/CMakeLists.txt:38 (ie_add_vs_version_file)
~~~

##### Steps to reproduce
~~~bash
git clone https://github.com/openvinotoolkit/openvino.git
git checkout tags/2021.4
git submodule update --init --recursive
mkdir build
cd build
cmake -DBUILD_SHARED_LIBS=OFF -DENABLE_VPU=OFF \
      -DENABLE_INTEL_GPU=OFF \
      -DENABLE_GNA=OFF \
      -DENABLE_MYRIAD=OFF \
      -DENABLE_HETERO=OFF \
      -DENABLE_MULTI=OFF \
      -DENABLE_TEMPLATE=OFF \
      -DENABLE_IR_V7_READER=OFF \
      -DNGRAPH_ONNX_FRONTEND_ENABLE=OFF \
      -DNGRAPH_PDPD_FRONTEND_ENABLE=OFF \
      -DNGRAPH_TF_FRONTEND_ENABLE=OFF \
      -DENABLE_GAPI_PREPROCESSING=OFF \
      -DENABLE_MKL_DNN=ON \
      -DNGRAPH_IR_FRONTEND_ENABLE=ON -G "Visual Studio 16 2019" -A x64 -DCMAKE_BUILD_TYPE=Release ..
~~~

This leads to the error mentioned above. Is there a bug in the tutorial?

##### Issue submission checklist

 - [X] I report the issue, it's not a question
   
 - [X] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
   
 - [X] There is reproducer code and related data files: images, videos, models, etc.

**Comment by jgespino** [comment]
Hi @dnns92 

The static library option was recently introduced and is not available in the 2021.4 branch. Please try to build using the latest master branch.

I tested on the latest master branch (e78ada117849c644f1c13f535e93cde970c45599) and didn't observe the error message. 

Regards,
Jesus

**Comment by dnns92** [comment]
okay thanks
